packages/public/server/src/module/Api/config/module.config.php

```diff
@@ -198,6 +207,15 @@
                                     ],
                                 ],
                             ],
+                            'event' => [
+                                'type' => 'segment',
+                                'options' => [
+                                    'route' => '/last-event',
+                                    'defaults' => [
+                                        'action' => 'lastEvent',
+                                    ],
+                                ],
+                            ],
```

Comment:
Fetches the last event -> it is used in the tests that a new event is created when a new comment is stored